package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "ohif-viewer",
-  "version": "0.0.20",
+  "version": "0.0.21",
```

Comment:
Can you remove this change? I believe the version of this library is updated automatically by semantic-release.